tests/integration/test_data/gomod_packages.yaml

```diff
@@ -14,11 +14,11 @@ without_deps:
     packages:
       - dependencies: []
         name: "github.com/cachito-testing/cachito-gomod-without-deps"
-        type: "gomod"
+        type: "go-package"
```

Comment:
All changes made to the `tests/integration/test_data` files are only sorting the packages/dependencies, matching them to the output produced while running the integration tests.

This allowed us to keep the sorting function as a private member of the Request model.